src/main.rs

```diff
@@ -38,9 +39,10 @@ use tao::menu::{MenuBar as Menu, MenuItem};
 
 mod state;
 
+
 static TOAST_MANAGER: AtomRef<ToastManager> = |_| ToastManager::default();
 static LANGUAGE: AtomRef<Language> = |_| Language::by_locale(AvailableLanguages::EnUS);
-static DEFAULT_PATH: Lazy<RwLock<PathBuf>> = Lazy::new(|| RwLock::new(PathBuf::from("./.warp")));
+static DEFAULT_PATH: Lazy<RwLock<PathBuf>> = Lazy::new(|| RwLock::new(PathBuf::from(dirs::home_dir().unwrap().join(".warp"))));
```

Comment:
You should be able to just do `RwLock::new(dirs::home_dir().unwrap().join(".warp"))` instead of keeping it inside `PathBuf::from` since `dirs::home_dir()` returns `Option<PathBuf>`.